Assets/MixedRealityToolkit/_Core/Inspectors/PropertyDrawers/MixedRealityPosePropertyDrawer.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Inspectors.PropertyDrawers
+{
+    [CustomPropertyDrawer(typeof(MixedRealityPose))]
+    public class MixedRealityPosePropertyDrawer : PropertyDrawer
+    {
+        private readonly GUIContent positionContent = new GUIContent("Position");
+        private readonly GUIContent rotationContent = new GUIContent("Rotation");
+        private Quaternion rotationCache = Quaternion.identity;
+        private Vector3 rotations = Vector3.zero;
+
+        public override float GetPropertyHeight(SerializedProperty property, GUIContent label)
+        {
+            return EditorGUIUtility.singleLineHeight * 3f;
```

Comment:
I presume the 3f is to indicate we need space for the title plus lines for each of the pose components. just in case we decide to change the layout (and to establish a solid pattern), should we, instead, define a constant so that when it is used later (ex: var fieldHeight = position.height / 3f;) we can skip the hard-coded constant?

Reply:
Unfortunately we have to set the property height here only per Unity.

Yes, it's space for the property name, position, and rotation lines.

Reply:
I'm open to moving the 3f to a local field and replacing it in the OnGUI below if that's what you're referring to.

Reply:
Given this is us configuring the editor layout to draw correctly, I see no reason to have this configurable.  It's not something a user would touch.